app/javascript/crayons/MentionAutocompleteTextArea/MentionAutocompleteTextArea.jsx

```diff
@@ -391,6 +392,7 @@ export const MentionAutocompleteTextArea = forwardRef(
                 <ComboboxList>
                   {users.map((user) => (
                     <ComboboxOption
+                      key={`${user.username}`}
```

Comment:
nitpick (not blocking) 
```suggestion
                      key={user.username}
```